signatures/Dilithium/dilithium/dilithium.yaml

```diff
@@ -0,0 +1,6 @@
+name: dilithium
+type: SIG
+security notion: SUF-CMA
+comment: >-
+  Note that Dilithium can be used as a deterministic or randomized signature
+  scheme.
```

Comment:
nope, all the same. The only difference is how the "randomness" is generated.

Reply:
I thought that's what flavours are for?

Reply:
You (FK) mean because the verification routines (and KAT values) do not differ, do they? That's indeed interesting, because this makes both compatible (because verification does not differ), however we cannot store the different randomness requirements with our data model...